app/src/main/res/layout/tab_tray_grid_item.xml

```diff
@@ -2,114 +2,146 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+
+<!--
+A FrameLayout here is an efficient way of having a views stack while allowing:
+    - a sufficient enough canvas for the play button to be outside translated
+    without it getting clipped by the GridLayoutManager
+    - enough padding for the primary view that needs to be displayed
+    where a border around it can be placed
+-->
+<FrameLayout
```

Comment:
Open question - I am curious if this needs to be a FrameLayout compare to a Linear or RelativeLayout? I don't really know the implications of having a FrameLayout nested inside of a FrameLayout compare to other Layouts.

Reply:
To me FrameLayout made more sense from a mental model, it basically allowing for a stack of:
- a big enough space for the thumbnail and where the play pause button to be translated
- a big enough space where an external border can be place
- then the item we want to show.
A ConstraintLayout or a even RelativeLayout (even with it's two measure passes) in place of the two FrameLayouts would probably get the same results with the same performance. I think the layout is pretty simple to see any performance hits by going either way.

Reply:
Refactored the original patch to use just one new layout as parent for the item, this being needed because of the exterior border and to allow for the play/pause button being outside translated.
So this should be as performant as possible.